recipe/test-jni.sh

```diff
@@ -1,20 +1,19 @@
 #!/bin/bash
-
 set -e
-export 
+
 if [ ! -d $JAVA_LD_LIBRARY_PATH ]; then
 	echo "Did you remember to activate the conda environment?"
 	exit -1
 fi
+
 os=$(uname -s | tr '[:upper:]' '[:lower:]')
+gcc -I$JAVA_HOME/include \
```

Comment:
We probably should add a compiler requirement in test and update this to `$CC`.

Reply:
Broke out in issue ( https://github.com/conda-forge/openjdk-feedstock/issues/41 ).